jbmc/regression/jbmc-cover/Makefile

```diff
@@ -4,9 +4,11 @@ include ../../src/config.inc
 
 test:
 	@../$(CPROVER_DIR)/regression/test.pl -p -c ../../../src/jbmc/jbmc
+        @../$(CPROVER_DIR)/regression/test.pl -p -c "../../../src/jbmc/jbmc --symex-driven-lazy-loading" -X symex-driven-lazy-loading-expected-failure -s symex-driven-loading
```

Comment:
@smowton, it seems you have replaced \t by spaces here and below...

Reply:
Yup. Fixed.